app/digests/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+DIGEST_ID_FORMAT = r'^[A-Za-z0-9\-._]+$'
+
+
+class Digest(models.Model):
+    id = models.CharField(primary_key=True, max_length=255)
+    content = JSONField()
+    image = JSONField()
+    impact_statement = models.TextField(blank=True, null=True)
+    published = models.DateTimeField()
+    related_content = JSONField()
```

Comment:
maybe logical order like in https://github.com/elifesciences/api-raml/blob/develop/src/snippets/digest.v1.yaml is easier to understand that alphabetical, but :man_shrugging:

Reply:
🤷‍♂️  though I will make sure the order in the response matches the schema.